config/opal_config_pthreads.m4

```diff
@@ -661,6 +665,10 @@ if test "$OMPI_TRY_FORTRAN_BINDINGS" = "$OMPI_FORTRAN_NO_BINDINGS" || \
   opal_pthread_fortran_success=1
 fi
 
+if test "$WANT_MPI_CXX_SUPPORT" -ne 1; then
```

Comment:
If you use the numeric test operators, you shouldn't need to use quotes on the variable under test (because it had better be guaranteed to be a number).